lib/osf-components/addon/components/registries/finalize-registration-modal/component.ts

```diff
@@ -44,13 +44,18 @@ export default class FinalizeRegisrationModalComponent extends Component {
     @computed('manager.draftManager.reviewsWorkflow')
```

Comment:
oh one thing just to be on the safe side.
```suggestion
    @computed('manager.draftManager.{reviewsWorkflow,draftRegistration.hasProject}')
```